.github/workflows/ci-build.yml

```diff
@@ -8,16 +8,19 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        docker_db_version: [ 7.0.10, 6.2.15 ]
+        docker_db_version: [ 6.2.17-d1, 7.0.13, 7.1.1 ]
```

Comment:
Is a matrix build really required here? 
I guess we don't have special behavior for certain versions here.

Reply:
But I know it's not part of this PR. Maybe we could add a ticket to remove it...

Reply:
I created ticket https://github.com/exasol/elasticsearch-virtual-schema/issues/47 for this.